src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -391,6 +397,13 @@ export class AppRoot extends polymerElementWithLocalize {
 
           <!-- Links section -->
           <paper-listbox>
+              <a
+                class="documentation-link"
+                hidden$="[[!shouldShowDocumentationLink]]"
+                href="https://www.reddit.com/r/outlinevpn/wiki/index/">
+                  <span>[[localize('outline-documentation')]]</span>
+                  <iron-icon icon="open-in-new" />
+              </a>
```

Comment:
*minor* I guess we should use a 2-space indentation here cuz the `<a>` should align with the `<span>` at line 407?

```suggestion
            <a
              class="documentation-link"
              hidden$="[[!shouldShowDocumentationLink]]"
              href="https://www.reddit.com/r/outlinevpn/wiki/index/">
              <span>[[localize('outline-documentation')]]</span>
              <iron-icon icon="open-in-new" />
            </a>
```

Reply:
Tis indeed: https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
I think we're talking about 2 different things. This whole block is indented by 4 instead of 2 spaces.

```
<paper-listbox>
    <a>
      ...
    </a>
  <span>...</span>
</paper-listbox>
```

It should be:

I think we're talking about 2 different things. This whole block is indented by 4 instead of 2 spaces.

```
<paper-listbox>
  <a>
    ...
  </a>
  <span>...</span>
</paper-listbox>
```

Reply:
Although now I'm questioning whether I misunderstood @jyyi1 😅

Reply:
> Oh i fixed that part already

Ok nvm me then. GitHub didn't show me the latest on this comment. Sorry!